test/performance/CMakeLists.txt

```diff
@@ -15,8 +15,14 @@ macro (seqan3_benchmark benchmark_cpp)
     seqan3_test_component (target "${benchmark}" TARGET_NAME)
     seqan3_test_component (test_name "${benchmark}" TEST_NAME)
 
+    # force alignment of benchmarked loops so that numbers are reliable:
```

Comment:
We don't have `[BENCH]`, only `[TEST]` in the commit message

Reply:
I addressed this.